src/client/Microsoft.Identity.Client/AuthScheme/PoP/PoPAuthenticationScheme.cs

```diff
@@ -71,19 +73,19 @@ public string FormatAccessToken(MsalAccessTokenCacheItem msalAccessTokenCacheIte
             var payload = new JObject(
                 new JProperty(PoPClaimTypes.At, msalAccessTokenCacheItem.Secret),
                 new JProperty(PoPClaimTypes.Ts, (long)(DateTime.UtcNow - s_jwtBaselineTime).TotalSeconds ),
-                new JProperty(PoPClaimTypes.HttpMethod, _httpRequestMessage.Method.ToString()),
-                new JProperty(PoPClaimTypes.Host, _httpRequestMessage.RequestUri.Host),
-                new JProperty(PoPClaimTypes.Path, _httpRequestMessage.RequestUri.AbsolutePath),
+                new JProperty(PoPClaimTypes.HttpMethod, _httpMethod.ToString()),
+                new JProperty(PoPClaimTypes.Host, _requestUri.Host),
+                new JProperty(PoPClaimTypes.Path, _requestUri.AbsolutePath),
                 new JProperty(PoPClaimTypes.Nonce, CreateNonce()),
                 new JProperty(PoPClaimTypes.Cnf, new JObject(
                     new JProperty(PoPClaimTypes.JWK, popAssertion))));
 
             string popToken =  CreateJWS(payload.ToString(Json.Formatting.None), header.ToString(Json.Formatting.None));
 
             // For POP, we can also update the HttpRequest with the authentication header
-            _httpRequestMessage.Headers.Authorization = new AuthenticationHeaderValue(
-                AuthorizationHeaderPrefix,
-                popToken);
+            //_httpRequestMessage.Headers.Authorization = new AuthenticationHeaderValue(
```

Comment:
I dont see where this object is being used after this is set. How are these headers used in the pop binding?